### PR TITLE
Combine Search Bars

### PR DESCRIPTION
**Pull request by justinjnlim**
Replaced multiple search bars with one search bar that will search based on the
structure of the input (glottocode formatting, etc.) but will still do partial
and non-English searches, unlike quicksearch.

Still needs unit tests in followup commit.

### CODE REVIEW

File: glottolog3/views.py [hunk 1]
```diff
@@ -261,11 +261,71 @@ def quicksearch(request):
...
+            'namequerytype': request.params['namequerytype'],
+            'multilingual': 'multilingual' in request.params
+            }
+
+    if not term:
+        query = None
+    elif len(term) < 3:
+        query = None
+        message = ('Please enter at least four characters for a name search '
+            'or three characters for an iso code')
```
Comment (baieric) [inline glottolog3/views.py]: this message should be changed to match the new behaviour

File: glottolog3/views.py [hunk 2]
```diff
@@ -261,11 +261,71 @@ def quicksearch(request):
...
+    elif len(term) == 8 and GLOTTOCODE_PATTERN.match(term):
+        query = query.filter(Languoid.id == term)
+        kind = 'Glottocode'
```
Comment (baieric) [inline glottolog3/views.py]: this case only works for when there's a full glottocode being searched, when we should support partial glottocode (e.g. `stan1`)

EDIT: nvm, this isn't a really important part of the spec and shouldn't block merging

**Review by baieric — CHANGES_REQUESTED** [review]
Looking good, just some small things that didn't match the spec.

I'd prefer we had unit tests as part of this PR, if it's possible to finish that before the demo tomorrow.

**Review by baieric — APPROVED** [review]

File: glottolog3/views.py (hunk 1 above)
Comment (justinjnlim) [reply glottolog3/views.py]: fixed

**Pull request merged**